src/actions/actionsAuth.ts

```diff
@@ -0,0 +1,38 @@
+import { logIn, getCurrentUser } from "@/firebase";
+
+export interface AuthActionProps {
+    request: Request;
+}
+
+export interface IUserData {
+    status: 'success' | 'error';
```

Comment:
you repeatedly used this exact strings to indicate statuses a few times in this file. 

put those statuses in the enum just to be safe.